config/action.d/badips.conf

```diff
@@ -10,7 +10,7 @@
 
 [Definition]
 
-actionban = curl --fail  --user-agent "fail2ban v0.8.12" http://www.badips.com/add/<category>/<ip>
+actionban = curl --fail  --user-agent "<agent>" http://www.badips.com/add/<category>/<ip>
```

Comment:
Well I don't like to use agent additionally to version in config reader:
- version is sufficient enough (agent can be and will be set in jail.conf);
- needed in some jails only (that will be not used from all the people);
- actions configs will not acquire this setting direct (added in jailreader currently), so it should be pass as parameter to corresponding action, and adding it in all readers will over-complicate a little bit, imho.